examples/bri-3/src/bri/identity/bpiSubjects/subjects.module.ts

```diff
@@ -8,25 +8,40 @@ import { GetAllBpiSubjectsQueryHandler } from './capabilities/getAllBpiSubjects/
 import { GetBpiSubjectByIdQueryHandler } from './capabilities/getBpiSubjectById/getBpiSubjectByIdQuery.handler';
 import { UpdateBpiSubjectCommandHandler } from './capabilities/updateBpiSubject/updateBpiSubjectCommand.handler';
 import { BpiSubjectStorageAgent } from './agents/bpiSubjectsStorage.agent';
+import { AccountController } from '../bpiAccounts/api/accounts.controller';
+import { CreateBpiAccountCommandHandler } from '../bpiAccounts/capabilities/createBpiAccount/createBpiAccountCommand.handler';
+import { UpdateBpiAccountCommandHandler } from '../bpiAccounts/capabilities/updateBpiAccount/updateBpiAccountCommand.handler';
+import { DeleteBpiAccountCommandHandler } from '../bpiAccounts/capabilities/deleteBpiAccount/deleteBpiAccountCommand.handler';
+import { GetAllBpiAccountsQueryHandler } from '../bpiAccounts/capabilities/getAllBpiAccounts/getAllBpiAccountQuery.handler';
+import { GetBpiAccountByIdQueryHandler } from '../bpiAccounts/capabilities/getBpiAccountById/getBpiAccountByIdQuery.handler';
+import { BpiAccountAgent } from '../bpiAccounts/agents/bpiAccounts.agent';
+import { BpiAccountStorageAgent } from '../bpiAccounts/agents/bpiAccountsStorage.agent';
 
 export const CommandHandlers = [
   CreateBpiSubjectCommandHandler,
   UpdateBpiSubjectCommandHandler,
   DeleteBpiSubjectCommandHandler,
+  CreateBpiAccountCommandHandler,
+  UpdateBpiAccountCommandHandler,
+  DeleteBpiAccountCommandHandler,
```

Comment:
@skosito What is the reasoning behind listing all of these handles and agents of the BpiAccount here in the subjects module rather than the accounts module?

Reply:
good point, i think i fixed that in other PR